skimage/measure/profile.py

```diff
@@ -14,10 +14,11 @@ def profile_line(image, src, dst, linewidth=1,
         (3D array, where the final axis contains the channel
         information).
     src : 2-tuple of numeric scalar (float or int)
-        The start point of the scan line.
+        The positive coordinates of the start point of the scan line.
```

Comment:
Is it worth adding this? An error message will be raised anyway otherwise. I'm all for clarity, but also short docstrings are easier to read.

Reply:
I will relax this requirement according to @stefanv https://github.com/scikit-image/scikit-image/issues/4279#issuecomment-580461567: negative values will be considered to be inside the the image (a warning will be raised anyway). Only out of range coordinates will cause an error to be raised.